src/wip/streaming-dca.rain

```diff
@@ -0,0 +1,188 @@
+              #Ideally set it 15 minutes ahead of the whatever the timestamp is while deploying
+              strat-strat-time: 1718738185
+                                
+              # Bounty amount denominated in input tokens
+              bounty-min: 0.02
+              bounty-max: 2
+              bounty-epoch-increase: 0.01
+              bounty-epoch: 1800
+    
+              # Uniswap v3 pool fees for input-output token pair
+              twap-io-fee: '[uniswap-v3-fee-low]'
+    
+              # Minimum input/output token ratio  
+              min-ratio: 0
+            scenarios:
+              prod:
+                bindings:
+                  plottables: '''plottables-prod'
+                  get-last-time: '''get-last-time-prod'
+              metric:
+                runs: 1
+                bindings:
+                  plottables: '''plottables-plot'
+                  get-last-time: '''get-last-time-plot'
+              
+
+charts:
+  red-streaming-dca:
+    scenario: streaming-dca.red.metric
+    metrics:
+      - label: RED Per second rate
+        value: 0.5.0
+        description: Amount of RED tokens sold per second.
+      - label: RED sold per hour
+        value: 0.5.1
+        description: Amount of RED tokens sold per hour.
+      - label: RED sold per day
+        value: 0.5.2
+        description: Amount of RED tokens sold per day.
+      - label: Minimum bounty
+        value: 0.5.3
+        description: Minimum bounty offered in BLUE tokens.
+      - label: Maximum bounty
+        value: 0.5.4
+        description: Maximum bounty offered in BLUE tokens.
+      - label: Bounty epoch
+        value: 0.5.5
+        description: Bounty epoch time in seconds
+      - label: Bounty epoch increase
+        value: 0.5.6
+        description: Increase in bounty per epoch
+      - label: Minimum ratio
+        value: 0.5.7
+        description: Maximum bounty offered in BLUE tokens.
+      
+                     
+---
+#input-token-address !Order input token address.
+#output-token-address !Order output token address.
+
+#twap-io-fee !Twap fee for the distribution token pool, paired with either stable token. 
+#min-ratio !Minimum input ouput ratio for the order.
+
+#bounty-min !Minimum bounty to offer for each trade in USD.
+#bounty-epoch-increase !Amount to increase bounty per bounty epoch.
+#bounty-epoch !Bounty epoch in seconds.
+#bounty-max !Maximum bounty that can be offered. 
+
+#per-second-rate !The rate of stable denominated distributed tokens distributed per second.
+#strat-strat-time !Strategy start timestamp
+#get-last-time !Binding to get last time and current time
+#plottables !Binding for charting metrics.
+
+#uniswap-words !The subparser for the Uniswap words
+#orderbook-subparser !The subparser for the Orderbook words
+#last-time-key "last-time-key"
+
+#bounty-auction
+  time-since-cooldown: ,
+  bounty: min(
+    linear-growth(
+      bounty-min
+      bounty-epoch-increase
```

Comment:
i thought we discussed in chat just having a single time which is the total auction duration?